packages/accounts-controller/src/AccountsController.ts

```diff
@@ -91,8 +91,7 @@ export type AccountsControllerActions =
   | AccountsControllerUpdateAccountsAction
   | AccountsControllerGetAccountByAddressAction
   | AccountsControllerGetSelectedAccountAction
-  | AccountsControllerGetAccountAction
-  | AllowedAccountsControllerActions;
```

Comment:
We don't want allowlists to be included in the controller actions/events unions. 
- This interferes with the type narrowing base-controller does to distinguish internal/external actions/events.
- This exposes allowlists externally since actions/events unions are exported at the package-level. We generally want allowlists to be scoped to its controller.